### PR TITLE
Seed files for all models

### PR DESCRIPTION
**Pull request by Squeech**
Seed file created and some seeds generated for DB

Mixture of Faker and manual seed creations done, all code in seed file

## Types of change

- [] New feature (change which adds functionality)

- [] Bug fix (change which fixes an issue)

- [] Improvement/refactoring (change that doesn't add any features but makes things better)

- [] Styling (style changes that don't affect functionality)

## Checklist before submitting

- [] Satisfies card on Trello

- [] Has had a another team mate review your design.

- [] No known defects

- [] Appropriate tests completed (e.g. core functionality, edge cases)

- [] Code has been formatted with nicely and contains proper indentation.

- [] Code comments as appropriate

- [] `master` branch has been merged into the feature branch and tested (if any newer commits on `master`)

### CODE REVIEW

File: app/models/booking.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Booking < ApplicationRecord
   validates :start_date, :end_date, presence: true
-  validates :end_date_after_start_date
+  # validates :end_date_after_start_date
```
Comment (learner1unknown) [inline app/models/booking.rb:3]: Why is this validation commented out ?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,38 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+# User.destroy_all
```
Comment (learner1unknown) [inline db/seeds.rb:9]: All the seed info is commented ?

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,38 @@
...
+#   arrival_address: Faker::Address.city,
+#   user_id: 11
+# )
+# puts "finished!"
+
+booking = Booking.create!(
+  start_date: DateTime.strptime("23/12/2022 11:00", "%d/%m/%Y %H:%M"),
+  end_date: DateTime.strptime("26/12/2022 11:00", "%d/%m/%Y %H:%M"),
+  total_price: (729*3),
+  user_id: 8,
```
Comment (learner1unknown) [inline db/seeds.rb:40]: don't hardcode user_id data, assign a user to variable then take that instance variable;s id. Same with aircraft id !

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,38 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
```
Comment (learner1unknown) [inline db/seeds.rb:6]: Delete unnecessary comments that are automatically generated with rails.

**Review by learner1unknown — CHANGES_REQUESTED** [review]
Hi @Squeech, Please see my comments throughout.

File: app/models/booking.rb (hunk 1 above)
Comment (Squeech) [reply app/models/booking.rb:3]: When I tried to run seed file without that commented, it came back with an error that said "Bookings require at least one validation" - after commenting that line it worked all fine, so i imagine the syntax may be wrong when calling validates on a custom method. Instead of looking into resolving this one I thought best to get the seeds done and move on as we can look at this custom validation later on or remove it entirely if we wish to do so.

File: db/seeds.rb (hunk 2 above)
Comment (Squeech) [reply db/seeds.rb:9]: I commented a lot of the seeds methods because I didn't know how to code it so it can create a new custom database each time assigning a aircraft to an existing user (or rather, a newly generated user) as well as create a booking between users and aircrafts. So I did each creation step of the database manually and commented the other methods out when running the seed file

File: db/seeds.rb (hunk 3 above)
Comment (Squeech) [reply db/seeds.rb:40]: If you can show me how to write the code that allows this to happen randomly when generating users on a new seed, would appreciate it!

File: db/seeds.rb (hunk 4 above)
Comment (Squeech) [reply db/seeds.rb:6]: Will do :)

File: db/seeds.rb (hunk 2 above)
Comment (learner1unknown) [reply db/seeds.rb:9]: I'll make some edits and we can chat on Thursday - what you have written is already good just the last booking bit!

File: app/models/booking.rb (hunk 1 above)
Comment (learner1unknown) [reply app/models/booking.rb:3]: Fair there probably isn't much value in stressing over these validations now - so we can delete it but I'll make a card on trello to capture it and it can be a task for later down the track

**Review by Squeech — COMMENTED** [review]
Incredibly work with that seed file!!!

**Review by drenetic — APPROVED** [review]

**Review by Zhenni-Z — APPROVED** [review]

**Review by drenetic — APPROVED** [review]

**Review by drenetic — APPROVED** [review]

**Review by learner1unknown — APPROVED** [review]
